### DISCUSSION TITLE
Image zoom when clicked?

### DISCUSSION THREAD

**Discussion by ledell** [h2oai/wave#1541] [Q&A] [answered]
On some websites, if you click on an image, it will blow up to fullscreen mode (with an "x" to exit the view).  Is there a way to do this in Wave?

**Comment by mturoci** [comment, marked as answer]
Hi Erin, this is currently not possible,  but feel free to file a feature request.

**Reply by ledell** [reply]
Thanks, will do @mturoci!

**Reply by ledell** [reply]
Feature request created here: https://github.com/h2oai/wave/issues/1542
